app/views/articles/show.html.erb

```diff
@@ -84,6 +84,7 @@
     class="container article"
     id="article-show-container"
     data-author-id="<%= @article.user_id %>"
+    data-author-class-name="<%= @article.user.class.name %>"
```

Comment:
I needed this value for my Javascript so I'm passing it in here

Reply:
This can be `data-author-class-name="User"` since the `@article.user` is always a user :)